substrate/bin/node-template/pallets/template/src/tests.rs

```diff
@@ -9,7 +9,7 @@ fn it_works_for_default_value() {
 		// Dispatch a signed extrinsic.
 		assert_ok!(TemplateModule::do_something(RuntimeOrigin::signed(1), 42));
 		// Read pallet storage and assert an expected result.
-		assert_eq!(TemplateModule::something(), Some(42));
+		assert_eq!(<Something<Test>>::get(), Some(42));
```

Comment:
Im not sure if we settled on a preferred syntax, this turbofish vs `::`.  
Personally i think `::` everywhere is better, but without linter it will probably not work out anyway.

Reply:
I agree that `Something::<Test>::get()` is preferred